README.md

```diff
@@ -60,4 +52,44 @@ dio.render(
 )
 ```
 
-The easiest way to get started with DIO is to read the [Introduction to DIO](https://dio.js.org/introduction.html) or look at the [API Documentation](https://dio.js.org/api.html).
+The easiest way to get started with DIO is to walk through the [Introduction to DIO](https://dio.js.org/introduction.html) or the [API Documentation](https://dio.js.org/api.html).
+
+## Highlights
+
+### Render
+
+1. Elements.
+2. Primitives: `Strings, Numbers, null, undefined`.
+3. Fragments: `Arrays, Iterables`.
+4. Async: `Promises`.
+6. Other: `Portals, Errors, Objects` etc.
```

Comment:
5* (and others, I tend to use `1` for every number instead of manually incrementing)

Reply:
Thanks, didn't know what was a thing.